custom_components/smaev/number.py

```diff
@@ -79,9 +79,28 @@ class SmaEvChargerNumberEntityDescription(NumberEntityDescription):
         native_unit_of_measurement=UnitOfEnergy.KILO_WATT_HOUR,
         entity_registry_enabled_default=True,
     ),
+    SmaEvChargerNumberEntityDescription(
+        key="charge_current_limit",
+        translation_key="charge_current_limit",
+        type=SMAEV_PARAMETER,
+        channel="Parameter.Inverter.AcALim",
+        native_step=1,
```

Comment:
Is it sufficient to have steps of 1A?

Reply:
IMHO that's right but I'll double check.

Reply:
Checked that 1A steps is correct.